internal/driver/tagroot.go

```diff
@@ -0,0 +1,117 @@
+package driver
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/google/pprof/profile"
+)
+
+// addLabelNodes adds psuedo-stack-frames "label:value" to each Sample with
```

Comment:
s/psuedo/pseudo/g

:)

Reply:
haha, thanks, let me grep for any more occurrences too